kustomize/overlays/sandbox/propeller/plugins/presto/propeller-patch.yaml

```diff
@@ -0,0 +1,20 @@
+# This file is only for volume mounts.  The configmap itself that's being mounted is sufficiently different that
```

Comment:
I wrote this comment at one point I think.  At the time, it made sense.  At this point, I have no idea what it means.